sycl/doc/extensions/proposed/sycl_ext_oneapi_device_global.asciidoc

```diff
@@ -24,15 +24,17 @@
 == Introduction
 In OpenCL 2.0 and later, a user is able to allocate program
 scope memory which can be accessed like a {cpp} global variable by any kernel in
-an OpenCL program (`cl_program`). When a program is shared between multiple devices,
-each device receives its own unique instance of the program scope memory allocation.
-
-This extension introduces device scoped memory allocations into SYCL that can be accessed
-within a kernel using syntax similar to {cpp} global variables, but that have unique
-instances per `sycl::device`. Mechanisms are provided for the host program to enqueue
-copies to or from the allocations on a specific device.  Restrictions are
-placed on the types of data that can be stored within `device_global` allocations, particularly
-around copyability and constructors/destructors.
+an OpenCL program (`cl_program`). When a program is shared between multiple
```

Comment:
Do I understand correctly that most of these changes are just re-formatting to fit into certain row lenght?

Reply:
I took the liberty of re-formatting the paragraphs I touched with these clarifications. As such all the paragraphs affected should also have some change related to the clarifications.